--- packages.x86_64_old	2025-03-19 12:22:55.000000000 +0100
+++ packages.x86_64_new	2025-11-12 17:31:28.971709172 +0100
@@ -114,6 +114,7 @@
 git
 glances
 python-packaging
+python-jinja
 gpm
 gptfdisk
 haveged
@@ -227,15 +228,17 @@
 kscreen
 kwallet-pam
 kwayland-integration
-libappindicator-gtk3
-maliit-keyboard
+kwin-x11
+libappindicator
 okular
 plasma-browser-integration
 plasma-desktop
 plasma-disks
 plasma-firewall
+plasma-keyboard
 plasma-nm
 plasma-pa
+plasma-x11-session
 plasma-systemmonitor
 plasma-workspace
 powerdevil
@@ -297,5 +300,4 @@
 ## VMware
 open-vm-tools
 xf86-input-vmmouse
-xf86-video-vmware
 xf86-video-qxl
