--- run_before_squashfs.sh	2023-02-12 23:07:56.396361665 +0100
+++ run_before_squashfs.sh.patched	2023-02-12 23:10:49.024387189 +0100
@@ -28,7 +28,7 @@
 # Init & Populate keys
 pacman-key --init
 pacman-key --populate archlinux endeavouros
-pacman -Sy
+pacman -Syy
 
 # Install liveuser skel (in case of conflicts use overwrite)
 pacman -U --noconfirm --overwrite "/etc/skel/.bash_profile","/etc/skel/.bashrc" -- "/root/endeavouros-skel-liveuser/"*".pkg.tar.zst"
@@ -86,11 +86,12 @@
 
 # Get extra drivers!
 mkdir "/opt/extra-drivers"
-sudo pacman -Sw --noconfirm --cachedir "/opt/extra-drivers" r8168
+pacman -Syy
+pacman -Sw --noconfirm --cachedir "/opt/extra-drivers" r8168
 
 # install packages
 mkdir -p "/usr/share/packages"
-sudo pacman -Sw --noconfirm --cachedir "/usr/share/packages" grub eos-dracut kernel-install-for-dracut refind os-prober xf86-video-intel
+pacman -Sw --noconfirm --cachedir "/usr/share/packages" grub eos-dracut kernel-install-for-dracut refind os-prober xf86-video-intel
 
 # Clean pacman log and package cache
 rm "/var/log/pacman.log"
