--- run_before_squashfs.sh	2025-03-19 12:22:55.000000000 +0100
+++ run_before_squashfs.sh.patched	2025-08-12 07:14:47.381197443 +0200
@@ -73,6 +73,9 @@
 pacman -U --noconfirm --needed -- "/root/packages/"*".pkg.tar.zst"
 rm -rf "/root/packages/"
 
+# reinstall eos-settings-plasma for offline installs:
+pacman -S --noconfirm eos-settings-plasma
+
 # Enable systemd services
 # --> now in airootfs/etc/systemd/system/multi-user.target.wants
 #systemctl enable NetworkManager.service systemd-timesyncd.service bluetooth.service firewalld.service
@@ -104,6 +107,8 @@
 # pacman -Scc seem to fail so:
 rm -rf "/var/cache/pacman/pkg/"
 
+# package changes for packagechoooser.conf (calamares)
+wget -O /etc/calamares/modules/packagechooser.conf "https://raw.githubusercontent.com/endeavouros-team/calamares/refs/heads/calamares/data/eos/modules/packagechooser.conf"
 
 #create package versions file
 pacman -Qs | grep "/calamares " | cut -c7- > iso_package_versions
