--- run_before_squashfs.sh	2025-03-27 20:17:41.870734336 +0100
+++ run_before_squashfs.sh.patched	2025-03-27 20:17:28.671735750 +0100
@@ -73,6 +73,9 @@
 pacman -U --noconfirm --needed -- "/root/packages/"*".pkg.tar.zst"
 rm -rf "/root/packages/"
 
+# reinstall eos-settings-plasma for offline installs:
+pacman -S --noconfirm eos-settings-plasma
+
 # Enable systemd services
 # --> now in airootfs/etc/systemd/system/multi-user.target.wants
 #systemctl enable NetworkManager.service systemd-timesyncd.service bluetooth.service firewalld.service
